package.json

```diff
@@ -18,6 +18,7 @@
     "release": "standard-version",
     "standard": "echo '\\033[1;33m  This project uses eslint instead of standard. Use `npm run lint` instead.'",
     "start": "ts-node -P ./tsconfig.json ./src/cli.ts",
+    "dev": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/cli.ts",
```

Comment:
if we already used ts-node, why nodemon instead of ts-node-dev? https://www.npmjs.com/package/ts-node-dev

Reply:
Ah I didn't know it existed!

Reply:
Good point, I didn't know it existed

Reply:
@eoln I might leave this for now - I tried switching it out, but `ts-node-dev` has an annoying "feature" where it rings the bell and sends a notification on every change... and this "feature" can't be switched off